queries/elixir.scm

```diff
@@ -0,0 +1,41 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(arguments) @arguments_list
+(call
+  target: (_) @functionCallee) @functionCall @functionCallee.domain
+
+;;(keywords
+;;  (pair
+;;    key: (_) @collectionKey @collectionItem.trailing.end.startOf
+;;    value: (_) @value) @collectionItem @collectionItem.trailing.start.endOf
+;;    (",")? @collectionItem.trailing.start.startOf
+;;    (#allow-multiple! @collectionItem))
+
+;; How does surrounding whitespace work for this
+(map_content
+  (binary_operator
+      left: (_) @collectionKey @collectionKey.trailing.start.endOf
+      right: (_) @value @collectionKey.trailing.end.startOf) @collectionItem
+      ","? @collectionItem.trailing
+      (#shrink-to-match! @collectionItem "([^,]+)"))
```

Comment:
What's this doing?

Reply:
I was trying to fix `item removal` being greedy, it's just from me still learning how the queries work. I see that a period does roughly what I was looking for, am I more on the right track here?:

```
(map_content
  (binary_operator
    left: (_) @collectionKey
    right: (_) @value
  ) @collectionItem
  .
  ","? @collectionItem.trailing
)
```

Reply:
ah yeah a `.` should fix that 👍